core/amber/src/main/scala/edu/uci/ics/amber/engine/common/AmberConfig.scala

```diff
@@ -105,4 +109,50 @@ object AmberConfig {
   // JDBC configuration
   val jdbcConfig: Config = getConfSource.getConfig("jdbc")
 
+  // ai assistant server configuration
+  val aiAssistant: String = getConfSource.getString("ai-assistant-server.default")
+
+  private val logger: Logger = Logger.getLogger(this.getClass.getName)
+
+  // this will return a boolean, in the code where you want to add AI feature,
+  // use "if (getAIAssistant())" to determine whether the AI feature will be exposed to the users.
+  def getAIAssistant(): Boolean = {
+    aiAssistant match {
+      case "none" =>
+        logger.warning("No AI Assistant")
+        false
+      case "openai" =>
+        val key: String = getConfSource.getString(("ai-assistant-server.account-key.openai-key"))
+        if (validateKey(key)) {
+          logger.info("The AI Assistant initialized successfully")
+          true
+        } else {
+          logger.warning("The OpenAI authentication key is not correct")
+          false
+        }
+      case _ =>
+        logger.warning("The OpenAI authentication key is not correct")
```

Comment:
I think this might not "key is not correct", let's give it a different log info.